pg2kinesis/formatter.py

```diff
@@ -11,6 +11,7 @@
 # Tuples representing changes as pulled from database
 Change = namedtuple('Change', 'xid, table, operation, pkey')
 FullChange = namedtuple('FullChange', 'xid, change')
+FullChange.operation = property(lambda self: self.change.get('kind'))
```

Comment:
Where is this used?

Reply:
This is used when determining what operation this change corresponds to, so we can decide whether it should be filtered out. The base `Change` object exposes this as `change.operation`, so this creates a shorthand for `full_change.operation`. 

This allows us to avoid something like:
```
if instanceof(change, Change):
    change.operation
elif instanceof(change, FullChange):
    change.change.get('kind')
else:
    ???
```
and instead just do `if change.operation in operations_to_replicate:...`.